kernel/aster-nix/src/thread/exception.rs

```diff
@@ -5,14 +5,14 @@
 use ostd::{cpu::*, mm::VmSpace};
 
 use crate::{
-    prelude::*, process::signal::signals::fault::FaultSignal,
+    arch::exception::log_trap_info, prelude::*, process::signal::signals::fault::FaultSignal,
     vm::page_fault_handler::PageFaultHandler,
 };
 
 /// We can't handle most exceptions, just send self a fault signal before return to user space.
 pub fn handle_exception(context: &UserContext) {
     let trap_info = context.trap_information();
-    let exception = CpuException::to_cpu_exception(trap_info.id as u16).unwrap();
+    let exception = trap_info.cpu_exception();
     log_trap_info(exception, trap_info);
```

Comment:
Since `trap_info` contains `CpuException`, the `log_trap_info` function only needs to take `trap_info` as its sole argument.